Cargo.toml

```diff
@@ -23,7 +23,7 @@ opt-level = 3
 [workspace.dependencies]
 serde = { version = "1.0" }
 strum = { version = "0.26", features = ["derive"] }
-
+image = "0.24"
```

Comment:
Let's depend on `0.25` since that's the version used by bevy and add a note to keep it in sync when updating bevy.
Currently, this adds a duplicated version for `image` to the dependency tree as shown in the `Cargo.lock` changelog which is not really necessary.